app/uk/gov/hmrc/testuser/models/TestUser.scala

```diff
@@ -74,14 +77,14 @@ case class TestAgent(override val userId: String,
   override val affinityGroup = "Agent"
 }
 
-case class TestIndividualCreatedResponse(userId: String, password: String, saUtr: Option[SaUtr], nino: Option[Nino], mtdItId: Option[MtdItId])
+case class TestIndividualCreatedResponse(userId: String, password: String, individualDetails: IndividualDetails, saUtr: Option[SaUtr], nino: Option[Nino], mtdItId: Option[MtdItId])
```

Comment:
Actually it will be used afterward to fetch individuals. though it could be re-added on the next story